ocaml/xenopsd/scripts/qemu-wrapper

```diff
@@ -148,6 +148,7 @@ def main(argv):
 
     # don't allow hvmloader to change the lower MMIO hole size
     xenstore_write("/local/domain/%d/hvmloader/allow-memory-relocate" % domid, "0")
+    xenstore_write("/local/domain/%d/hvmloader/pci/xen-platform-pci-bar-uc" % domid, "0")
```

Comment:
Could you add a comment above? Now it looks like it's part of the MMIO hole thing.